cmd/apiserver_receive_adapter/main.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2019 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+
+	"os"
+
+	"github.com/knative/eventing/pkg/adapter/apiserver"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
```

Comment:
I thought I could get away with this since this belongs to the receiver only. I'm fine removing the dependency on controller-runtime and migrate to `pkg/controller`

Out of curiosity, what the rationale behind the decision of getting rid of controller-runtime?

Reply:
https://github.com/knative/eventing/issues/734